commit c5693827412eace5806f8b7d469f7188cdf95c99 (refs/changes/77/4677/3)
Author: sjorge <sjorge@blackdot.be>
Date:   2018-08-17T16:48:31+00:00 (1 year, 2 months ago)
    
    OS-7116 bhyve brand should accept bios and uefi values for bootrom (fix cstyle)
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index 1e2c6771be..7c6b84f1ef 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -424,7 +424,7 @@ add_lpc(int *argc, char **argv)
 				val = DEFAULT_BOOTROM_CSM;
 			} else if (strcmp(val, "uefi") == 0) {
 				val = DEFAULT_BOOTROM;
-                        }
+			}
 		}
 		if (snprintf(conf, sizeof (conf), "%s,%s", lpcdevs[i], val) >=
 		    sizeof (conf)) {
